toolchain/check/cpp/import.cpp

```diff
@@ -1694,6 +1694,13 @@ auto ImportCppFunctionDecl(Context& context, SemIR::LocId loc_id,
 
   SemIR::Function& function_info = context.functions().Get(*function_id);
   if (IsCppThunkRequired(context, function_info)) {
+    Diagnostics::AnnotationScope annotate_diagnostics(
+        &context.emitter(), [&](auto& builder) {
+          CARBON_DIAGNOSTIC(InCppThunk, Note,
+                            "in thunk for Cpp function used here");
```

Comment:
I think we backtick `Cpp` elsewhere, like `InCppNameLookup` and `CppInteropMissingLibrary` diagnostics
```suggestion
                            "in thunk for `Cpp` function used here");
```

Reply:
Makes sense. (This was copied from the old `InCallToCppFunction` diagnostic which didn't do that.) But how would you feel about switching to "C++" instead of "Cpp"? This isn't talking about the Cpp package.

Reply:
I think either would be fine; C++ function because it's C++, or `` `Cpp` `` function because it'll show up in the `Cpp` package, right? (I only think "Cpp function" is a bit odd)

Reply:
Changed to C++.